MyWeight/Localization/Localization.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Localization.swift
+//  MyWeight
+//
+//  Created by Bruno Koga on 20/06/16.
+//  Copyright © 2016 Diogo Tridapalli. All rights reserved.
+//
+
+import Foundation
+
+struct Localization {
```

Comment:
I'm not 100% sure that creating a struct with static funcs with all the `NSLocalizedString`s in one place is the best solution here. But I **think** I prefer this than having `NSLocalizedString`s all over the place.